### ISSUE TITLE
Implement Analytics

### ISSUE THREAD

**Issue by charparr** [ua-snap/data-api#51]
NOT Google. Vacuum logs to 3rd party from EB instance?
See https://github.com/citruspi/Flask-Analytics

**Comment by brucecrevensten** [comment]
Let's figure out if we care about this one?  On the one hand, seeing API queries gives us a nice way to see which places are being queried.  We probably don't care a lot about users browsing the HTTP interface (that's a navigational aid, not intended to be crucial content that we'd care about page counts, etc).

I think my vote is to close it.  @BobTorgerson @charparr if you agree, let's close this!

**Comment by charparr** [comment]
Yeah I vote to close as well, this doesn't feel like a must have for me. And, I'm not sure what we'd do with the information if we had it anyway. "Wow, 1000 people queried Kotzebue in the last year" would be cool information to see, but I'm not sure it helps us.
